core/packages/contracts/test/BeefyClient.t.sol

```diff
@@ -112,88 +112,88 @@ contract BeefyClientTest is Test {
-
-    function testSubmitFailForPrevRandaoTooEarlyOrTooLate() public {
-        initialize(setId);
-        beefyClient.submitInitial(commitHash, bitfield);
-        // reverted for commit PrevRandao too early
-        cheats.expectRevert(BeefyClient.WaitPeriodNotOver.selector);
-        beefyClient.commitPrevRandao(commitHash);
-
-        // reverted for commit PrevRandao too late
-        cheats.roll(block.number + randaoCommitDelay + randaoCommitExpiration + 1);
-        cheats.expectRevert(BeefyClient.TicketExpired.selector);
-        beefyClient.commitPrevRandao(commitHash);
-    }
-
-    function testSubmitWithHandover() public {
-        //initialize with previous set
-        initialize(setId - 1);
-
-        beefyClient.submitInitialWithHandover(commitHash, bitfield);
-
-        cheats.roll(block.number + randaoCommitDelay);
-
-        cheats.difficulty(difficulty);
-
-        beefyClient.commitPrevRandao(commitHash);
-
-        BeefyClient.Commitment memory commitment = BeefyClient.Commitment(blockNumber, setId, payload);
-        beefyClient.submitFinalWithHandover(
-            commitment, bitfield, finalValidatorProofs, mmrLeaf, mmrLeafProofs, leafProofOrder
-        );
-        assertEq(beefyClient.latestBeefyBlock(), blockNumber);
-    }
-
-    function testScaleEncodeCommit() public {
-        BeefyClient.Payload memory _payload = BeefyClient.Payload(
-            0x3ac49cd24778522203e8bf40a4712ea3f07c3803bbd638cb53ebb3564ec13e8c, hex"0861620c0001026d6880", hex""
-        );
-        BeefyClient.Commitment memory _commitment = BeefyClient.Commitment(5, 7, _payload);
-        bytes memory encoded = beefyClient.encodeCommitment_public(_commitment);
-        assertEq(
-            encoded,
-            hex"0861620c0001026d68803ac49cd24778522203e8bf40a4712ea3f07c3803bbd638cb53ebb3564ec13e8c050000000700000000000000"
-        );
-    }
+//    function testSubmitFailWithStaleCommitment() public {
+//        // first round of submit should be fine
+//        testSubmit();
+//
+//        beefyClient.submitInitial(commitHash, bitfield);
+//        cheats.roll(block.number + randaoCommitDelay);
+//        cheats.difficulty(difficulty);
+//        beefyClient.commitPrevRandao(commitHash);
+//        BeefyClient.Commitment memory commitment = BeefyClient.Commitment(blockNumber, setId, payload);
+//        //submit again will be reverted with StaleCommitment
+//        cheats.expectRevert(BeefyClient.StaleCommitment.selector);
+//        beefyClient.submitFinal(commitment, bitfield, finalValidatorProofs);
+//    }
+//
+//    function testSubmitFailWithInvalidBitfield() public {
+//        initialize(setId);
+//
+//        beefyClient.submitInitial(commitHash, bitfield);
+//
+//        cheats.roll(block.number + randaoCommitDelay);
+//
+//        cheats.difficulty(difficulty);
+//
+//        beefyClient.commitPrevRandao(commitHash);
+//
+//        BeefyClient.Commitment memory commitment = BeefyClient.Commitment(blockNumber, setId, payload);
```

Comment:
Are comments here for test CI breaking? Add some analysis here
https://snowfork.slack.com/archives/G01BGMGLAC9/p1683682087326629?thread_ts=1683621046.426719&cid=G01BGMGLAC9

Reply:
Exactly, I was trying to figure out what was breaking but the Github incidents yesterday interrupted me. I'll revert the changes - best way forward is using Nix in CI to lock all of our dependencies.